### PR TITLE
fix #795

### LINKED ISSUE #795: bars and punctum mora
This is a first step to #767 

Gregorio used to do that so I'm labelling this issue "bug", although it might have been removed on purpose. The idea is to align bars in the middle of notes, not taking into account the punctum mora.

Current gregorio behavior for

```
Ió(g.) (,) i(g) (,) i(g)
```

is

mora-bar

while the first bar should be moved left from (with of punctum mora + width between punctum mora and note).

### PR DESCRIPTION
**Pull request by eroux**
test PR coming

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ As of v3.0.0 this project adheres to [Semantic Versioning](http://semver.org/).
 - Headers not recognized by gregorio, rather than inducing an error, are now simply accepted by gregorio.  Most of the old "standard" headers, like `book` and `manuscript-location`, are now handled in this way and thus are no longer limited to one instance.  See GregorioRef for details.
 - `mode` will now be converted to Roman numerals in lower case so that it shows up as small capitals using the default `modeline` style.  See [UPGRADE.md](UPGRADE.md) for details (for the change request, see [#756](https://github.com/gregorio-project/gregorio/issues/756)).
 - `\grecommentary` is now less restrictive about where it occurs and need not occur directly before the score anymore.
+- When a bar is preceeded by a punctum mora, gregorio now ignores the punctum mora in the bar horizontal placement by default, and also adds a custom space (defaulting to 0). You can change this behavior with `\gresetbarshiftaftermora{}`, see GregorioRef for its arguments.
```
Comment (henryso) [inline CHANGELOG.md]: This entry should have a link to the issue.

Comment (eroux) [inline CHANGELOG.md]: done, thanks!

**Comment by eroux** [comment]
See https://github.com/gregorio-project/gregorio-test/pull/106 for tests

**Comment by henryso** [comment]
Looks good.

**Pull request merged**